### ISSUE TITLE
Config option to remove TODO and Example in migration file?

### ISSUE THREAD

**Issue by aswinr2c** [seppevs/migrate-mongo#171]
Is there a way to remove the comments (the TODO and Example) in every single migration?

```
module.exports = {
  async up(db, client) {
    // TODO write your migration here.  <=== Getting rid of this part
    // See https://github.com/seppevs/migrate-mongo/#creating-a-new-migration-script
    // Example:
    // await db.collection('albums').updateOne({artist: 'The Beatles'}, {$set: {blacklisted: true}});
  },

  async down(db, client) {
    // TODO write the statements to rollback your migration (if possible)  <=== Getting rid of this part
    // Example:
    // await db.collection('albums').updateOne({artist: 'The Beatles'}, {$set: {blacklisted: false}});
  }
};

```

**Comment by TeNNoX** [comment]
See my PR #74 - already provides a way to do this.

**Comment by seppevs** [comment]
I've merged PR #74, so you can now override the sample migration. Closing this.
